store/index.js

```diff
@@ -532,8 +533,14 @@ export const mutations = {
             p_state.dataDictionary.annotated[columnName].valueMap = {};
         }
 
-        // 1. Assign the option value to a raw value for this column
-        p_state.dataDictionary.annotated[columnName].valueMap[rawValue] = optionValue;
+        if (optionValue === null) {
```

Comment:
It would be good to just add a brief comment here explaining the if/else statement